### PR TITLE
[ACA-2087] Overlay Viewer

### PR DESCRIPTION
**Pull request by pionnegru**
Issue Number: [ACA-2087](https://issues.alfresco.com/jira/browse/ACA-2087)

## Changes
change preview router with the viewer outlet

### CODE REVIEW

File: src/app/services/content-management.service.ts [hunk 1]
```diff
@@ -614,6 +616,7 @@ export class ContentManagementService {
       .subscribe(
         () => {
           this.store.dispatch(new ReloadDocumentListAction());
+          this.store.dispatch(new ClosePreviewAction());
```
Comment (suzanadirla) [inline src/app/services/content-management.service.ts]: this is done on `undoMoveNodes`; shouldn't be also added for `moveNodes`?

File: protractor.conf.js [hunk 2]
```diff
@@ -104,7 +104,8 @@ exports.config = {
         '--headless',
         '--remote-debugging-port=9222',
         '--disable-gpu',
-        '--no-sandbox'
+        '--no-sandbox',
+        '--disable-dev-shm-usage'
```
Comment (adinapitul) [inline protractor.conf.js]: can you please revert this change?

File: src/app/components/layout/app-layout/app-layout.component.ts [hunk 3]
```diff
@@ -137,18 +137,9 @@ export class AppLayoutComponent implements OnInit, OnDestroy {
...
-            event instanceof NavigationStart &&
-            // search employs reuse route strategy
-            !event.url.startsWith('/search;')
-          );
-        }),
-        takeUntil(this.onDestroy$)
-      )
-      .subscribe(() => this.store.dispatch(new SetSelectedNodesAction([])));
+    this.router.events.pipe(takeUntil(this.onDestroy$)).subscribe(() => {
+      this.store.dispatch(new ResetSelectionAction());
```
Comment (suzanadirla) [inline src/app/components/layout/app-layout/app-layout.component.ts]: Do we need to reset selection on each of the router events? We should filter the router events like it was before
